### ISSUE TITLE
Known Issues (KSP induced)

### ISSUE THREAD

**Issue by shadowmage45** [shadowmage45/TUFX#14]
Below is a list of issues that are mostly inherent with the way KSP is using the Unity game engine, and are not problems that I am interested in solving, or simply cannot solve without breaking KSP.
* HDR can sometimes have some strange transparency artifacts, especially on stock part shaders.  Unknown the exact cause, but parts using TU's PBR shaders work fine.  (this used to manifest on the stock buildings as well)
* HDR can cause some other occasional camera-angle dependent artifacts, some truly bizarre stuff
* Banding with ambient occlusion when camera close to an object.  Caused by the insane far-clip settings used by KSP cameras.

**Comment by JonnyOThan** [comment]
Current theory on the "transparency" issue is that when HDR is enabled, the alpha channel exceeds 1.0 which causes the underlying colors to come out inverted.  In this image, the blue sky becomes orange inside the fairing, the dark flagpole and trim around the cockpit come out light, and the white parts of the cockpit come out dark.

![image](https://user-images.githubusercontent.com/9804078/172932236-c1577b57-2e2f-400c-8995-7500bb6b89c6.png)

As for a fix...it would probably require patching/replacing all the stock shaders.

**Comment by JonnyOThan** [comment]
Textures Unlimited already has the ability to replace the shaders used for various models.  I successfully did a simple test of the mobile processing lab, where previously I could see the ladder parts on the other side of it.

The other test case I was using (and where I notice this problem most often) is with fairings.  But since the fairing is not a standard model, I don't think TexturesUnlimited can patch it (but I only just started using TU an hour ago, so maybe I'm missing something).  It would also be prohibitive to list EVERY model in the game for TU to patch, and wouldn't help non-stock parts that use the broken stock shaders.

It *may* be possible to create a mod that uses TU and entirely replaces the stock shaders with TU's versions at runtime, which would *just fix* this problem for everyone.
